packages/account/src/Components/unlink-modal/__tests__/unlink-modal.spec.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { UnlinkModal } from '../unlink-modal';
+
+const modalRoot = document.createElement('div');
+modalRoot.setAttribute('id', 'modal_root');
+document.body.appendChild(modalRoot);
+
+describe('<UnlinkModal />', () => {
+    it('should show the proper messages', () => {
+        render(<UnlinkModal identifier_title={'title'} is_open />);
```

Comment:
render(<UnlinkModal identifier_title="title" is_open />);

Reply:
Thanks for notice it, I will fix